quantum/quantum_keycodes.h

```diff
@@ -123,6 +123,11 @@ enum quantum_keycodes {
     KC_LEAD,
 #endif
 
+#ifdef COMPOSE_ONBOARD_ENABLE
+    KC_COMPOSE_ONBOARD,
+    KC_COMPOB = KC_COMPOSE_ONBOARD,
+#endif
+
```

Comment:
1. The `KC_COMPOSE_ONBOARD` keycode should be added near the end of the `quantum_keycodes` enum, just before `SAFE_RANGE` on Line 520, so as not to break compatibility with [VIA Configurator](https://caniusevia.com/).
2 The `KC_COMPOB` alias should be created using a `#define` statement around Line 814, as `#define KC_CMOB KC_COMPOSE_ONBOARD` if you've applied the suggestion I gave above.